### PR TITLE
Instructor training form text 

### PR DESCRIPTION
**Pull request by maneesha**
This addresses #2096 by updating the text at the top of the instructor training form.

### CODE REVIEW

File: amy/templates/forms/trainingrequest.html [hunk 1]
```diff
@@ -8,52 +8,19 @@
...
-teaching, and how The Carpentries can help in your community.
-Spaces are limited, so use this form to share with us what excites you
-about spreading our teaching to your organization. We will notify
-applicants as limited spaces become available. If you have any
-questions, please mail <a href="mailto:team@carpentries.org">
-team@carpentries.org</a>.</p>
+<b>For trainees selected by Carpentries member organizations or groups that have arranged for training</b>, it serves as a trainee <b>Profile Creation form</b>. This creates an entry in our database and allows us to record attendance and track your progress through the Instructor certification process. Form responses are not reviewed in detail but may be used in programmatic assessment. Please note that this form is separate from event registration. Your site contact will provide you with instructions to register for an Instructor Training event. In response to the first question, please select <i>Profile Creation for Pre-approved Trainees</i> and enter in your provided registration code.  If you do not have a registration code, please enter the name of the institution you are joining through.</p>
```
Comment (pbanaszkiewicz) [inline amy/templates/forms/trainingrequest.html:15]: `<p>` missing at the beginning of this paragraph.

File: amy/templates/forms/trainingrequest.html [hunk 2]
```diff
@@ -8,52 +8,19 @@
 
 {% block content %}
 
-<p class="lead">
-  Our instructor training classes are for people who wish to learn how to
-  teach Carpentry workshops. We teach impactful pedagogical techniques and
-  prepare you to become an Instructor in our community. We use this form to
-  register interest and build new communities of Instructors around the world.
+<p>
+  This form serves as the entry point for Instructor Training for two groups:
```
Comment (pbanaszkiewicz) [inline amy/templates/forms/trainingrequest.html:12]: You removed `class="lead"`, which rendered
> Our instructor training classes are for people who wish to learn how to...

in slightly bigger font. Are you sure you don't want users to highlight on this first paragraph?

**Review by pbanaszkiewicz — CHANGES_REQUESTED** [review]
Minor issue found in HTML.

File: amy/templates/forms/trainingrequest.html (hunk 1 above)
Comment (maneesha) [reply amy/templates/forms/trainingrequest.html]: Fixed

File: amy/templates/forms/trainingrequest.html (hunk 2 above)
Comment (maneesha) [reply amy/templates/forms/trainingrequest.html:12]: Correct - since we have changed the content of this text, we no longer need a section highlighted.

**Review by pbanaszkiewicz — APPROVED** [review]

**Pull request merged**